tests/rptest/services/kgo_verifier_services.py

```diff
@@ -442,14 +444,24 @@ def _ingest_status(self, worker_statuses):
         self._parent._status = reduced
 
     def poll_status(self):
+        retry_strategy = Retry(total=5,
+                               connect=5,
+                               read=5,
+                               backoff_factor=1,
```

Comment:
That's 30s worth of retry. I'd lower to something like 10s at most but fine either way. I.e. a lower backoff factor.

```
backoff_factor ([float](https://docs.python.org/3/library/functions.html#float)) –
A backoff factor to apply between attempts after the second try (most errors are resolved immediately by a second try without a delay). urllib3 will sleep for:

{backoff factor} * (2 ** ({number of previous retries}))
```